typescript/packages/subsurface-viewer/src/layers/wells/wellsLayer.ts

```diff
@@ -578,15 +611,19 @@ export default class WellsLayer extends CompositeLayer<WellsLayerProps> {
             })
         );
 
-        return [
-            outline,
-            log_layer,
-            colors,
-            highlight,
-            highlightMultiWells,
-            selection_layer,
-            names,
-        ];
+        if (this.props.optimizedRendering && this.props.optimizedInteraction) {
+            return [colors] as unknown as LayersList;
```

Comment:
colors as layer list ?

Reply:
Its not the colors but the list. That is not "colors" but "[colors]"

Reply:
Ok I see that lint no longer complains about that line (it did!)  so I removed the type cast

Reply:
Agree on that: I havent changed existing names but should probably do it later.